libbeat/docs/release-notes/5.0.0.asciidoc

```diff
@@ -0,0 +1,268 @@
+[[release-notes-5.0.0]]
+=== 5.0.0 Combined Release Notes
+
+The list below covers changes between 5.0.0-alpha1 to 5.0.0-GA.
+
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+- Run function to start a Beat now returns an error instead of directly exiting. {pull}771[771]
+- The method signature of `HandleFlags()` was changed to allow returning an error {pull}1249[1249]
+- Require braces for environment variable expansion in config files {pull}1304[1304]
+- On DEB/RPM installations, the binary files are now found under `/usr/share/{{beat_name}}/bin`, not in `/usr/bin`. {pull}1385[1385]
+- The logs are written by default to self rotating files, instead of syslog. {pull}1371[1371]
+- Remove deprecated `host` option from elasticsearch, logstash and redis outputs. {pull}1474[1474]
+- All configuration settings under `shipper:` are moved to be top level configuration settings. I.e.
+  `shipper.name:` becomes `name:` in the configuration file. {pull}1570[1570]
+- The `topology_expire` option of the Elasticserach output was removed. {pull}1907[1907]
+- Rename the `filters` section to `processors`. {pull}1944[1944]
+- Introduce the condition with `when` in the processor configuration. {pull}1949[1949]
```

Comment:
Generic filtering is part of Beats starting with 1.1.0: https://github.com/elastic/beats/pull/1120